dev/build_release.py

```diff
@@ -529,6 +511,14 @@ def init_argument_parser(cls, parser):
         '--bintray_repo', default='',
         help='Publish to this bintray repo.\n'
              'This requires BINTRAY_USER and BINTRAY_KEY are set.')
+      parser.add_argument(
```

Comment:
I could be confused.

If I am building jars I put them in this repository. Fine. But I dont build every jar I depend on, only the ones defined in this [git] repository. I dont know how this works under the covers, but isnt clear how the precedence and scoping works (or how things get removed so that I can use official external jars even if I may have built those projects myself earlier so the same versions are in my repo (built were built with a different tool chain).